### DISCUSSION TITLE
Is v-scope needed on explicit mount targets?

### DISCUSSION THREAD

**Discussion by TechAkayy** [vuejs/petite-vue#227] [Q&A]
Hi all, 

When mounting an app to a explicit target, is `v-scope` required on this target? 

```
createApp().mount('#only-this-div')

...
...

<div id="only-this-div" v-scope>
  <p>{{ count }}</p>
  <p>{{ plusOne }}</p>
  <button @click="increment">increment</button>
</div>
```

It's not clear in the readme, and also I noticed in this vueschool [article](https://vueschool.io/articles/vuejs-tutorials/building-a-multi-step-form-with-petite-vue/) about petite-vue advising not to miss the `v-scope` attribute even when mounting to a specific target. Can anyone validate this? Thanks!

![image](https://github.com/vuejs/petite-vue/assets/73772701/b163546a-f0eb-40ed-80ba-89bae13abf8e)

**Comment by sqllyw** [comment]
I think petite-vue will default to the root if no v-scope is found, but it gives a warning , so better use one:
~~~~
   if (!roots.length) {
        roots = [el]
      }

      if (
        import.meta.env.DEV &&
        roots.length === 1 &&
        roots[0] === document.documentElement
      ) {
        console.warn(
          `Mounting on documentElement - this is non-optimal as petite-vue ` +
            `will be forced to crawl the entire page's DOM. ` +
            `Consider explicitly marking elements controlled by petite-vue ` +
            `with \`v-scope\`.`
        )
      }

~~~~
